packages/hashtower.sol/package.json

```diff
@@ -0,0 +1,50 @@
+{
+    "name": "hashtower.sol",
+    "private": true,
+    "scripts": {
+        "start": "hardhat node",
+        "build": "hardhat compile",
+        "compile": "hardhat compile",
+        "deploy:test-contracts": "hardhat deploy:tree-contracts",
+        "test": "hardhat test",
+        "test:report-gas": "REPORT_GAS=true hardhat test",
+        "test:coverage": "hardhat coverage",
+        "lint": "solhint 'contracts/**/*.sol'"
+    },
+    "devDependencies": {
+        "@ethersproject/providers": "^5.4.7",
+        "@nomicfoundation/hardhat-chai-matchers": "^1.0.0",
+        "@nomicfoundation/hardhat-network-helpers": "^1.0.0",
+        "@nomicfoundation/hardhat-toolbox": "^2.0.2",
+        "@nomiclabs/hardhat-ethers": "^2.0.0",
+        "@nomiclabs/hardhat-etherscan": "^3.0.0",
+        "@typechain/ethers-v5": "^10.1.0",
+        "@typechain/hardhat": "^6.1.2",
+        "@types/chai": "^4.2.0",
+        "@types/mocha": ">=9.1.0",
+        "circomlibjs": "^0.0.8",
+        "dotenv": "^14.3.2",
+        "ethers": "^5.4.7",
+        "hardhat": "^2.13.0",
+        "hardhat-gas-reporter": "^1.0.8",
+        "prettier-plugin-solidity": "^1.0.0-beta.19",
+        "solhint": "^3.3.6",
+        "solhint-plugin-prettier": "^0.0.5",
+        "solidity-coverage": "^0.8.1",
+        "typechain": "^8.1.0"
+    },
+    "dependencies": {
+        "poseidon-solidity": "0.0.3"
+    },
```

Comment:
Would you like to add the authors of this package?

Reply:
Added. (in `contracts/package.json`)